packages/office-addin-sso/package.json

```diff
@@ -36,6 +36,7 @@
     "morgan": "1.9.1",
     "node-fetch": "^2.6.1",
     "office-addin-usage-data": "^1.6.14",
+    "path-to-regexp": "^0.1.12",
```

Comment:
I don't think we need to explicitly list the dependency here.  I think fixing the "express" entry that has a dependency on this should be sufficient.

Reply:
These changes are following the steps outlined internally to address this vulnerability.